build.sbt

```diff
@@ -26,26 +29,45 @@ version in Docker := "1.0"
 
 organization := "com.codacy"
 
-val brakemanVersion = "3.3.0"
+lazy val toolVersion = TaskKey[String]("Retrieve the version of the underlying tool from patterns.json")
```

Comment:
This is wrong, the correct syntax is `lazy val toolVersion = TaskKey[String]("toolVersion", "Retrieve the version of the underlying tool from patterns.json")`. I had some problems and this fixed it.